lib/css/components/post-summary.less

```diff
@@ -309,38 +305,94 @@
 
 .s-post-summary__watched {
     background-color: var(--yellow-050);
-}
 
-.s-post-summary__deleted {
-    background-color: var(--red-050);
+    .s-post-summary--content-title > a {
+        color: var(--blue-700);
 
-    .is-deleted {
-        color: @white;
-        background-color: var(--red-500);
+        &:hover,
+        &:active,
+        &:visited {
+            color: var(--blue-700);
+        }
+    }
```

Comment:
Did you intend for the watched post summary title anchor to have the same color regardless of state?

Reply:
It looks like active & hover are the same, and can be the same color as the title link color which is blue-700. I checked to make sure that's ok for accessibility, as long as the hover state has the icon. 

For visited though, that can probably go one darker to blue-800 to follow what visited states look like on other styles.